changelog/1.25.0.md

```diff
@@ -3,6 +3,13 @@ title: "1.25.0"
 description: "Released on 11/17/2021"
 ---
 
+The final patch release of Kubernetes 1.19 was published on October 28, 2021. As
```

Comment:
Not sure the best way to indicate this, maybe we need a new "Notices" section for advance warning about future changes?

Reply:
I think a notes callout is fine?

Reply:
What do you mean? Would you mind editing this directly?

Reply:
oops -- it would be helpful if I pushed my changes :)

Reply:
Looks good to me, thanks! Merging :)